### DISCUSSION TITLE
compiling issue? console.log() next.js not working - stumped...

### DISCUSSION THREAD

**Discussion by JonathanCamberos** [vercel/next.js#68332] [Help]
### Summary

https://github.com/JonathanCamberos/MyPortfolio.git   <--- for git clone to see if you can recreate the bug

first next.js application, testing console.log() not functioning 
have a console.log() set to print on page loading, however its not printing server side on the console that ran "npm run dev"
 a bit stumped as to why

code:
![image](https://github.com/user-attachments/assets/aeb9c28d-dd76-4870-b535-6d4acf1cfa58)

my console result:
![image](https://github.com/user-attachments/assets/fb43dee6-2156-4adc-bfe8-2df5e88ab046)

**but simply compiles, "/page" and does not print on the last line of console**
**the tutorial im following results in:**

expected result:
![image](https://github.com/user-attachments/assets/e576d029-7fb1-4b9a-b38c-757148feefd0)

**the main difference i think?**
**for the tutorial it results in "compiled client and server successfully" wereas for mine its simply compiling the "/page" itself, hence the server side console.log() isnt working.**

Its my first project so not sure what to do from here, any help is much appreciated!

**Comment by JonathanCamberos** [comment]
failed solution: deleting cache via deleting .next folder and running "npm run dev" again

**Reply by mohidmakhdoomi** [reply]
See if its printing in the browser console? might be logging it on client instead of server.

**Reply by mohidmakhdoomi** [reply]
I see, i'm also relatively new to next js but try running `next build` and then `next start` to have it run like it would it production. `next dev` behaves differently.

**Reply by JonathanCamberos** [reply]
neither sadly 

![image](https://github.com/user-attachments/assets/f77bbfc6-776b-4d58-babd-5237b2239e9e)

**Comment by icyJoseph** [comment]
Hi,

The issue is that you are silencing the logs... where did you get this codebase?

See here, https://github.com/JonathanCamberos/MyPortfolio/blob/main/next.config.js

```js
/** @type {import('next').NextConfig} */

const {withContentlayer} = require("next-contentlayer")

const nextConfig = {
    compiler:{
        removeConsole: true, // ❗️❗️❗️❗️❗️ this removes the logging!
    }
};

module.exports = withContentlayer({ ...nextConfig });
```
